ddtrace/llmobs/_llmobs.py

```diff
@@ -364,6 +364,7 @@ def enable(
         # override the default _instance with a new tracer
         cls._instance = cls(tracer=_tracer)
         cls.enabled = True
+        config._llmobs_enabled = True
```

Comment:
we should respect the env var if it's set, only doing this if there is no value

Reply:
Yup, we do that already immediately on enable() a few lines [up](https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/llmobs/_llmobs.py#L318-L320), so we don't get to this point if someone sets  `DD_LLMOBS_ENABLED=False`